workflow-templates/im-deploy-az-app-manually.yml

```diff
@@ -103,27 +110,71 @@ jobs:
           scope: ${{ env.ENVIRONMENT }}
           create-output-variables: true
         env:
-          # This variable is used to swap slots and annotate app insights
+          # This variable is used to swap slots
           RESOURCE_GROUP@dev: ''
           RESOURCE_GROUP@qa: ''
           RESOURCE_GROUP@stage: ''
+          RESOURCE_GROUP@stage-secondary: ''
           RESOURCE_GROUP@demo: ''
           RESOURCE_GROUP@uat: ''
           RESOURCE_GROUP@prod: ''
+          RESOURCE_GROUP@prod-secondary: ''
+
+          # Primary resource group used to pull azure credentials and secrets from
+          PRIMARY_RESOURCE_GROUP@dev: ''
+          PRIMARY_RESOURCE_GROUP@qa: ''
+          PRIMARY_RESOURCE_GROUP@stage stage-secondary: ''
+          PRIMARY_RESOURCE_GROUP@demo: ''
+          PRIMARY_RESOURCE_GROUP@prod prod-secondary: ''
+
           # This variable is used to deploy the app, swap slots, annotate app insights and send a teams notification
           AZ_APP_NAME@dev: ''
           AZ_APP_NAME@qa: ''
           AZ_APP_NAME@stage: ''
+          AZ_APP_NAME@stage-secondary: ''
           AZ_APP_NAME@demo: ''
           AZ_APP_NAME@uat: ''
           AZ_APP_NAME@prod: ''
+          AZ_APP_NAME@prod-secondary: ''
+          # TODO: Uncomment if you don't use an MSI with your app service
```

Comment:
That comment seems backward.... shouldn't it be "Uncomment if you _do_ use an MSI?"

And in fact, I can't think of any case where a repository that's prepared for this kind of deployment to a secondary region for failover _wouldn't_ use user-assigned MSIs, so probably everything about MSIs should _not_ be commented-out in this template.

Reply:
Fixed it.  Yes I originally had delete if you don't use it.